cluster/gce/configure-vm.sh

```diff
@@ -851,6 +851,17 @@ function node-docker-opts() {
   if [[ -n "${EXTRA_DOCKER_OPTS-}" ]]; then
     DOCKER_OPTS="${DOCKER_OPTS:-} ${EXTRA_DOCKER_OPTS}"
   fi
+
+  # Decide whether to enable the cache. Both of these variables
+  # come from the "kube-env" metadata value.
+  if [[ "${ENABLE_DOCKER_REGISTRY_CACHE:-}" == "true" && -n "${ZONE:-}" ]]; then
+    REGION=$(echo "${ZONE}" | cut -f 1 -d -)
+    if [[ "${REGION}" == "europe" ]]; then
+      REGION="eu"
+    fi
```

Comment:
Right now, this annoying logic is captured in the deployment scripts in https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/util.sh. I'd prefer the `kube-env` variable just derive from that, and be in the same place in the deployment code. An option here is just for it to be `DOCKER_REGISTRY_CACHE_REGION`, and if it's not present, it's off.